packages/mobile/src/components/api-rewards-drawer/ApiRewardsDrawer.tsx

```diff
@@ -58,7 +58,7 @@ const createStyles = (themeColors: ThemeColors) =>
     }
   })
 
-const ApiRewardsDrawer = () => {
+export const ApiRewardsDrawer = () => {
```

Comment:
OOC, why do we prefer named exports over default exports? Saw that rule in the Stems repo as welll

Reply:
I've been meaning to add this to the style guide: https://www.notion.so/audiusproject/Client-Javascript-Styleguide-4f34640139fc4a809c3765131bc597ff#55771014f786447890727d940eb3838a